js/core/utils/position.js

```diff
@@ -34,7 +39,29 @@ const getBoundingRect = (element) => {
     return rect;
 };
 
+const getAvailableBoundsContainer = (container) => {
+    const $container = container && $(container);
```

Comment:
To my mind, it's good idea to resolve case when passed `container` is not defined at the test beginning to simplify code below